src/generator_rom.rs

```diff
@@ -0,0 +1,55 @@
+// the generator ROM from:
+// https://github.com/Chia-Network/chia-blockchain/blob/main/chia/wallet/puzzles/rom_bootstrap_generator.clvm.hex
+pub const GENERATOR_ROM: [u8; 737] = [
```

Comment:
What about using https://doc.rust-lang.org/std/macro.include_bytes.html and creating a small binary file with the generator ROM?

Reply:
Or even just including it as hex, like we did in python. (Requires `hex` library.)

Reply:
including it as hex would require making it a `lazy_static!`, right?

Reply:
Maybe not... not sure if the compiler is smart enough to evaluate it as a constant at compile time. In any case, `include_bytes!` wouldn't need `lazy_static!` for sure.

Reply:
I don't think this blob will be here more than 6 months or so (perhaps I'm optimistic). We'll probably need it for the tests going forward though